tests/functional/test_firefox_dnt.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
```

Comment:
I recommend adhering to PEP8 for Python code. For imports standard libraries, external libraries, and internal libraries should be separated by a blank line. See more about imports here: http://legacy.python.org/dev/peps/pep-0008/#imports